client/my-sites/hosting/staging-site-card/index.js

```diff
@@ -186,16 +187,19 @@ export const StagingSiteCard = ( { currentUserId, disabled, siteId, siteOwnerId,
 	const getManageStagingSiteContent = () => {
 		return (
 			<>
-				<p>
-					{ translate( 'Your staging site is available at {{a}}%(stagingSiteName)s{{/a}}.', {
-						args: {
-							stagingSiteName: stagingSite.url,
-						},
-						components: {
-							a: <a href={ stagingSite.url } />,
-						},
-					} ) }
-				</p>
+				<p>{ translate( 'Your staging site is available at:' ) }</p>
+				<div style={ { display: 'flex', alignItems: 'center', marginBottom: 24 } }>
+					<SiteIcon siteId={ stagingSite.id } size={ 40 } style={ { marginLeft: '10px' } } />
+					<div
+						className="site__info"
+						style={ { display: 'flex', flexDirection: 'column', marginLeft: 10 } }
+					>
```

Comment:
Same here, we could use a styled component instead of an inline style.

We could also remove the className `site__info` and add `flex-shrink: 0` in the `.site-icon in SiteRow`.